server/src/main/java/org/elasticsearch/index/mapper/AbstractGeometryFieldMapper.java

```diff
@@ -136,6 +139,14 @@ protected Object parseSourceValue(Object value) {
                 }
             };
         }
+
+        @Override
+        public BlockLoader blockLoader(BlockLoaderContext blContext) {
```

Comment:
Can we move this to AbstractPointGeometryFieldMapper and remove the override method in AbstractShapeGeometryFieldMapper?

Reply:
There is no intermediate class common between Geo and Cartesian points, so that would require quite a bit of restructuring. Besides, the plan is to support geo_shape very soon, so we would just move back to this then anyway.

Reply:
I thought `AbstractPointGeometryFieldMapper` is common for geo and cartesian points. I know it will change but we should keep PRs to the minimum please or it makes reviewing very difficult.